README.md

```diff
@@ -4,11 +4,11 @@ A Serilog provider for [Microsoft.Extensions.Logging](https://www.nuget.org/pack
 
 ### ASP.NET Core Instructions
 
-**ASP.NET Core** applications should prefer [Serilog.AspNetCore](https://github.com/serilog/serilog-aspnetcore) and `UseSerilog()` instead.
+**ASP.NET Core** applications should prefer [Serilog.AspNetCore](https://github.com/serilog/serilog-aspnetcore#instructions) and `AddSerilog()` instead.
```

Comment:
I updated the links to go directly to the Instructions section of each repo, but happy to revert that change if you would still prefer them to just be to the repos.